scripts/maintenance/migrate_project/src/r_clone.py

```diff
@@ -0,0 +1,78 @@
+from pathlib import Path
+from subprocess import PIPE, Popen
+
+DESTINATION = "dst"
+SOURCE = "src"
+
+CONFIG = """
+[{destination}]
+type = s3
+provider = AWS
```

Comment:
see before

Reply:
This will change, I will generate it on the fly for source and detention an no longer will use hardcoded configuration.